### ISSUE TITLE
GGMAX Bug - Landscape issue that also messes with water

### ISSUE THREAD

**Issue by ZD-ZMA** [TheGameCreators/GameGuruRepo#5613]
![Screenshot (289)](https://github.com/TheGameCreators/GameGuruRepo/assets/124307560/804f21f3-6931-48d1-af72-f44770146507)
![Screenshot (290)](https://github.com/TheGameCreators/GameGuruRepo/assets/124307560/1714a0d0-734f-419d-9a5e-2f3a936fd68e)
![Screenshot (291)](https://github.com/TheGameCreators/GameGuruRepo/assets/124307560/64c4d501-ece9-418f-a372-2846f5a14f7d)
![Screenshot (292)](https://github.com/TheGameCreators/GameGuruRepo/assets/124307560/80310e10-4cd1-405f-8a5b-5a36d4b9a689)
![Screenshot (293)](https://github.com/TheGameCreators/GameGuruRepo/assets/124307560/a1fa0cc7-6432-4d1d-a8d8-e2029c6841f0)
all i did was make a new plains level. i tried to use builder editor but the builder editor came up black so i restarted ggmax and the this happened to the level...

**Comment by Kasseyus** [comment]
I’m currently in the process of going through all the outstanding issues.  As this ticket is over a month old, I'm just checking to see if you are still having a problem with this, or if the matter has since been resolved, either through an update to the engine, or if the issue was resolved when another, similar issue was fixed.

Please let me know by replying to this issue and I will investigate further.

**Comment by ZD-ZMA** [comment]
sorry. i was homeless recently so i wasnt able to get back to you. this issue that i reported is a rare bug. doesnt always happen and i dont know what triggers it, but its daintily happened multiple times. ive never reported it before because of how rare the bug happened. but an entire project got hit with the bug at once and i had to start over.

**Comment by Kasseyus** [comment]
@ZD-ZMA Sorry to hear that, that's pretty rough.  Hope you get back on your feet soon!

As to this bug, it's not an easy one to diagnose due to the rarity, but there are steps we can follow to try and figure out the cause.

Firstly, MAX creates a series of backups for your levels so if it happens, so you should not loose all of your work.  It's also a good practice (not just on MAX, for any sort of development) to keep a backup of your backups.

Secondly, what are your system specifications, and are you running any of the beta versions (DevTest/Experimental) or the Standard version?

Thirdly, can you remember any commonalities from when this bug presented itself in the past, anything consistent.  For example, you mention this happened when creating a new plains level, and used the BE.  Was one or both of those actions involved in previous occurrences of this bug.

Finally, if it happens again, if you are able to take a video of any part of the process, that may be useful.

**Comment by ZD-ZMA** [comment]
it happened again 
![Screenshot (445)](https://github.com/user-attachments/assets/8bbfd9af-c95e-4153-8871-6fecc66d41a4)

**Comment by ZD-ZMA** [comment]
i just got the cold war pack and while i was working on it, this happened

**Comment by Kasseyus** [comment]
The last time it happened - with the Cold War Pack - was that also while you was trying to use the Building Editor?

**Comment by ZD-ZMA** [comment]
> The last time it happened - with the Cold War Pack - was that also while you was trying to use the Building Editor?

no. its happened before i ever had cold war pack as well. i haven't used the builder editor in what seems like forever because of a bug that i reported that hasn't been fixed. a separate one. (black screen... same one other people had issues with)

this issue has been happening even when i dont do anything except only mess with the land scape. example being raise or flatten. thats it. i dont have to do anything else. i dont add anything... nothing.

**Comment by LeeBamberTGC** [comment]
@ZD-ZMA Can you attach the level that is shown in the screenshot, it looks like I am seeing the edge of the terrain but as we know MAX terrain is procedural and infinite, there is no edge, unless heightmaps are being used.  Once I have the FPM level for the above, I should know more :)

**Comment by LeeBamberTGC** [comment]
NOTE: Just going through the older 2024 bug reports marked WAITING FOR MORE INFO.  Feel free to add new info here using the latest DEVTEST build. If nothing new is posted, I will assume it cannot be reproduced with the newer updates and close this one in the near future.

**Comment by ZD-ZMA** [comment]
> NOTE: Just going through the older 2024 bug reports marked WAITING FOR MORE INFO. Feel free to add new info here using the latest DEVTEST build. If nothing new is posted, I will assume it cannot be reproduced with the newer updates and close this one in the near future.

go ahead and close it as i cannot recreate it currently. if in the future it happens again, ill try to make sure to record a clip of how i did it.
